app-office/gnumeric/files/gnumeric-1.12.52-slibtool.patch

```diff
@@ -0,0 +1,35 @@
+From 3031cc85718dde12a34be46330ef3b8c885f4f9e Mon Sep 17 00:00:00 2001
```

Comment:
Add link to PR at top (I know it's in commit message, but I like them being in the patches too, to ease lookups if we ever wonder "ok, what's the status of this?")

Reply:
Good idea, done.